pkg/baseline/packageIndex.json

```diff
@@ -1,127 +1,133 @@
 {
-  "Packages": {
-    "_Comments": {
-      "_StableVersions": [
-        "Released versions."
-      ],
-      "_BaselineVersion": [
-        "Used to determine the minimum version used for all packages in the repo."
-      ],
-      "_AssemblyVersionInPackageVersion": [
-        "Used to understand what version of a package to reference.",
-        "Then raise the package version of the reference so that its at least the baseline version."
-      ]
+    "Packages": {
+        "_Comments": {
+            "_StableVersions": [
+                "Released versions."
+            ],
+            "_BaselineVersion": [
+                "Used to determine the minimum version used for all packages in the repo."
+            ],
+            "_AssemblyVersionInPackageVersion": [
+                "Used to understand what version of a package to reference.",
+                "Then raise the package version of the reference so that its at least the baseline version."
+            ]
+        },
+        "svcutilcore": {
+            "InboxOn": {},
+            "AssemblyVersionInPackageVersion": {
+                "1.0.0.0": "4.5.0"
```

Comment:
We didn't release svcutilcore as a stable v4.5.0 package yet. We should not add this section at this point.